### DISCUSSION TITLE
Correct ports for SearXNG searches?

### DISCUSSION THREAD

**Discussion by n4tur4lb0rnch1ll4** [searxng/searxng#3332] [Q&A]
Hello everyone,

kind of new to SearXNG - I'm hosting myself on a NAS and want to restrict the firewall but I cannot make it work with the ports - I run into timeouts in all search engines. I tried opening up 80, 8080, 8888, 443 and some others but to no avail.

Can somebody please support me on this issue?

Thank you very much in advance!

**Comment by return42** [comment]
Sorry I can't help you on this concrete topic .. may someone other .. but if you do not get a feedback here I recommend to visit our community at Matrix --> [#searxng:matrix.org](https://matrix.to/#/#searxng:matrix.org)
